ldap/servers/slapd/libglobs.c

```diff
@@ -6811,6 +6848,19 @@ config_get_referral_mode(void)
     return ret;
 }
 
+int
+config_get_num_listeners(void)
+{
+    slapdFrontendConfig_t *slapdFrontendConfig = getFrontendConfig();
+    int retVal;
+
+    CFG_LOCK_READ(slapdFrontendConfig);
+    retVal = slapdFrontendConfig->num_listeners;
```

Comment:
This needs to check for num_listeners > 0

Reply:
@Firstyear, as the default value is > 0 and the setter checks that the value is between 1 or 4, 
I do not think we really need to add a check here, especially when none of the other getter functions have such tests